packages/react/src/conditional-field-container/docs/overview.mdx

```diff
@@ -0,0 +1,226 @@
+---
+title: Conditional field container
+description: A standardised pattern for conditionally hiding and revealing the amount of content a user views in a form.
+group: Forms
+storybookPath: /story/Forms-ConditionalFieldContainer--basic
+unreleased: true
+relatedPatterns: ['conditional-reveal']
+---
+
+This component aims to standardise the styling and behavior of the [conditional reveal pattern](/patterns/conditional-reveal) in the Design System.
+
+The conditional field container component wraps the children elements in a [FormStack](/components/form-stack).
```

Comment:
This sentence is weird. Took me a while to parse, but I think this is what we're getting at in a succinct manner
```suggestion
The conditional field container component wraps its children in a [FormStack](/components/form-stack).
```